tests/test_dimension.py

```diff
@@ -1370,7 +1370,7 @@ def test_topofusion_w_subdims_conddims(self):
         eqns = [Eq(f.forward, f + 1),
                 Eq(g.forward, g + 1),
                 Eq(fsave, f.dt2, implicit_dims=[ctime]),
-                Eq(h, f + g, subdomain=grid.interior),
+                Eq(h, f.dx + g, subdomain=grid.interior),
```

Comment:
Why? Does this break with the new additions?

Reply:
no, of course it doesn't "break" anything (why would it), but it's necessary to generate loop blocking due to these improvements: https://github.com/devitocodes/devito/pull/2016/files#diff-9a7b7fd6b3b90ced519c980132e20df6dabab8f794895fe9237f60880c80f006R98-R105

the alternative would have been to change the asserts. Perhaps equivalent.

Reply:
By "break", I mean the test would not pass